### PR TITLE
Changed the command call line in windows

### PR DESCRIPTION
**Pull request by dibyadas**
"os.system('cmd /c '+ new)" to  os.system('cmd /c ' +'"' +new+'"')

### CODE REVIEW

File: play.py [hunk 1]
```diff
@@ -29,7 +29,7 @@
     # Initialize the dict with files > 10MB in size
     for vid in files:
         # Remove files of size less than 10 MB (Subtitles, etc.)
-        if os.stat(vid).st_size < 10e7:
```
Comment (OrkoHunter) [inline play.py:32]: Does this make any difference in Python on Windows?

**Comment by dibyadas** [comment]
Yes . It seems so in python windows. 10e7 equals 1 with 8 zeroes. In our context it means 100MB.  But we want 1e7 which equals 10 MB

**Comment by dibyadas** [comment]
And I don't know why I can't reply to your comment. I using my mobile.

**Comment by OrkoHunter** [comment]
Alright

**Pull request merged**